tests/ops/test_helpers/vault_client.py

```diff
@@ -1,13 +1,11 @@
-import logging
 import os
 from typing import Any, Dict, Optional
 
 from hvac import Client
+from loguru import logger
```

Comment:
this was the only logging reference i did not get rid of in `tests`. the reason is because [here](https://github.com/ethyca/fides/blob/main/tests/ops/test_helpers/vault_client.py#L44) we actually are catching an exception that occurs on many of our tests, i.e it's an expected state in many of our test runs. we log that exception, but i wasn't seeing it at all in any of my test runs locally, so i'm not sure it was ever getting through. i did switch it to use `loguru` and still did not see the message, but figured i'd keep it here since we do need to catch the exception anyway